XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/UnityInput/GenericJoystickController.cs

```diff
@@ -138,7 +141,8 @@ protected void UpdateSingleAxisData(MixedRealityInteractionMapping interactionMa
                 case DeviceInputType.Trigger:
                 case DeviceInputType.TriggerPress:
                 case DeviceInputType.TouchpadPress:
-                    interactionMapping.BoolData = interactionMapping.FloatData.Equals(interactionMapping.InvertXAxis ? -1f : 1f);
+                    var processor = interactionMapping.GetInputProcessors<InvertSingleAxisProcessor>();
+                    interactionMapping.BoolData = interactionMapping.FloatData.Equals(processor.Invert ? -1f : 1f);
```

Comment:
Still don't like how we're doing this. Still wanna try to find a way to understand when this flag should be set without having to know about the processor from outside the interaction mapping.  Maybe somehow set that flag internally?